app/controllers/admin/events_controller.rb

```diff
@@ -141,7 +141,7 @@ def restart
     def vote
       @votes = @event.votes.includes(:user)
 
-      if (votes = current_user.votes.find_by_event_id(params[:id]))
+      if (votes = current_user.votes.find_by(votable_type: 'Event', votable_id: params[:id]))
```

Comment:
just `votable: @event` should also work, here and in other places where you query in a similar way. Give it a shot and let me know :)

Reply:
Yes it does, thanks :smile: